Source/WSTagsField.swift

```diff
@@ -565,13 +588,37 @@ open class WSTagsField: UIView {
             else {
                 frame.size.height = oldContentHeight
             }
+            
+            self.textfieldValidation = self.validateTagsInputTextField(forValue: text)
```

Comment:
isn't the assigning of this variable part of the validateTagsInputTextField function?